cmd/aries-js-worker/index.html

```diff
@@ -106,7 +106,7 @@
             document.getElementById("mode-label").innerHTML = `Running in ${mode} mode`
 
             if (src == "rest"){
-                document.getElementById("opts").innerHTML = `{"assetsPath": "/dist/assets", "agent-rest-url": "http://localhost:8082"}`
+                document.getElementById("opts").innerHTML = `{"assetsPath": "/dist/assets", "agent-rest-url": "http://localhost:8082", "agent-rest-wshook":"ws://localhost:8881"}`
```

Comment:
the rest code called it a notifier rather than a hook.

Reply:
in agent-rest start command we call it webhook, so reused that term here.
`notifier` term is use for "startNotifier/stopNotifier" where we watch for incoming messages.

Reply:
Looking at these URLs, I see that we should document somewhere how to properly run this `index.html`

Reply:
ie. this goes way past just inflating the wasm file. It requires services up and running in addition to the inflated wasm.

Reply:
one other point: can we start using property names with no special characters such that the double quotes aren't needed? Like `assetsPath` for instance.

Reply:
updated docs

> This new option requires a line in the docs